physical/raft/raft.go

```diff
@@ -130,6 +130,13 @@ type LeaderJoinInfo struct {
 	// via 'leader_api_addr'.
 	AutoJoin string `json:"auto_join"`
 
+	// Scheme defines the optional URI protocol scheme for addresses discovered via
+	// auto-join.
+	Scheme string `json:"scheme"`
```

Comment:
Do we want to include AutoJoin in the name to make explicit what it's for?

Reply:
Yeah I was thinking we should.